source/releasenotes/2024-11-12-content-publisher-beta.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Explore Pantheon Content Publisher—available now in Beta!
+published_date: "2024-11-12"
+categories: [New Feature, WordPress, Plugins, Drupal]
+---
+
+With Pantheon Content Publisher, your team can now publish content directly from Google Docs to Drupal, WordPress, or Next.js sites—quickly and seamlessly. Skip the extra steps and empower your team to keep content fresh and connected across platforms.
```

Comment:
Is emdash usage normal on the docs / release notes?

Reply:
The emdashes are used correctly in this release notes - they're here to set off additional information. No spaces around emdashes. Do you find them odd?